### PR TITLE
Update footer board info

### PR DESCRIPTION
**Pull request by lserranoruber**
Changed the names from CheckOUT members to fOUTje members and added board pic

### CODE REVIEW

File: outsite.nl/assets/images/photos/boards/board23.jpg
Comment (casperboone) [inline outsite.nl/assets/images/photos/boards/board23.jpg:1]: Could you please throw this pic in tinyjpg.com to make it a bit smaller? 😊

**Review by casperboone — COMMENTED** [review]
Looks good! Minor thingy

File: outsite.nl/assets/images/photos/boards/board23.jpg
Comment (lserranoruber) [reply outsite.nl/assets/images/photos/boards/board23.jpg:1]: this is after passing it through tinyjpg like 5 times 🥲 I'll throw it in a few more times but I don't think it'll get much smaller than 500kb

**Review by thomcsmits — COMMENTED** [review]
It makes sense to met that vicevoorzitter would be below voorzitter in the list, do you agree?

Also note that in Dutch, it's more commonly vicevoorzitter (as 1 word) (though both are accepted)

**Comment by lserranoruber** [comment]
@thomcsmits: good points, I changed around the order so it makes more sense, and fixed the vicevoorzitter as one word.
@casperboone: resized the image to 332kb

**Pull request merged**